docker/liberica-base/Dockerfile

```diff
@@ -34,5 +34,11 @@ RUN ln -s /lib /lib64 \
   &&   ${GLIBC_PREFIX}/sbin/ldconfig \
   &&   echo 'hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4' > /etc/nsswitch.conf \
   &&   apk add --no-cache jattach --repository http://dl-cdn.alpinelinux.org/alpine/edge/community/ \
-  &&   apk add --no-cache tini
-
+  &&   apk add --no-cache tini libstdc++ \
```

Comment:
Oh no, not `libstdc++`! I am quite worried about the amount of bloat in these images, seeing as the main aim of maintaining our own base images was to ensure they are as lean as possible. I know it's a lot of work, but I think this might work better as a separate image or extra build step on the image produced by `sbt-native-packager`

Reply:
AFAICT libstdc++ is ~1.7MB which seems to be about 1% of the total disk usage for one of our small apps. Personally I'm quite happy to pay that kind of price to be able to profile on-going problems without having to redeploy and throw away the problematic state. What do you think?

Reply:
Yeah for sure 👍